test/blocks/marketo/marketo.test.js

```diff
@@ -42,6 +43,41 @@ describe('marketo', () => {
   });
 });
 
+describe('marketo setProductOfInterest', () => {
+  beforeEach(() => {
+    sinon.stub(sessionStorage, 'getItem');
```

Comment:
since wtr runs everything in a browser, is there a reason you are not just using sessionStorage?

Reply:
True, updated